releases/2.13.14.md

```diff
@@ -0,0 +1,54 @@
+# Scala 2.13.14 (DRAFT)
+
+The Scala team at Lightbend is pleased to announce Scala 2.13.14.
+
+The following changes are highlights of this release:
+
+### Align with Scala 3
+
+* Introduce `-Xsource-features`, for customizing the behavior of `-Xsource:3` and `-Xsource:3-cross` ([#10709](https://github.com/scala/scala/pull/10709) by [@som-snytt](https://github.com/som-snytt))
```

Comment:
This should say lrytz. (The comment on Mar 12 explains.) If anything, I just slowed him down.

Reply:
We like people that slow others down in Switzerland (no irony!), that's the core of our political system :)